romana-install/romana-setup

```diff
@@ -25,7 +25,7 @@ usage() {
 	echo "Platforms:   aws (default), vagrant"
 	echo "Distro:      ubuntu (default), centos"
 	echo "Stack Types: devstack (default), kubernetes"
-	echo "Actions:     install (default), uninstall"
+	echo "Actions:     install (default), uninstall, test"
```

Comment:
So, "test" assumes that a cluster has been successfully installed and then runs whatever tests we have against the existing cluster?

Could you add an example line for that, please?